ocs_ci/ocs/fiojob.py

```diff
@@ -548,6 +551,10 @@ def is_storage_reclaimed():
         # Without checksum, we just need to make sure that data were deleted
         # and wait for this to happen to avoid conflicts with tests executed
         # right after this one.
-        delete_fio_data(fio_job_file, is_storage_reclaimed)
+        if delete_fio_data_after_test:
+            delete_fio_data(fio_job_file, is_storage_reclaimed)
+        # Wait until the end of the tests, and then delete the data.
+        else:
+            request.addfinalizer(lambda: delete_fio_data(fio_job_file, is_storage_reclaimed))
```

Comment:
I haven't seen `addfinalizer` being used not in fixtures before. Is this a correct usage?

Reply:
Here are 2 examples that I can see in 'tests/conftest' file: 
https://github.com/red-hat-storage/ocs-ci/blob/master/tests/conftest.py#L833
https://github.com/red-hat-storage/ocs-ci/blob/master/tests/conftest.py#L901

Reply:
ack, thanks for checking